tools/ci/ci_tools_unittest.sh

```diff
@@ -5,21 +5,33 @@ SCRIPT_DIR=$(dirname $(readlink -f "$0"))
 WPT_ROOT=$(readlink -f $SCRIPT_DIR/../..)
 cd $WPT_ROOT
 
+run_applicable_tox () {
+    # instead of just running TOXENV (e.g., py27)
+    # run all environments that start with TOXENV
+    # (e.g., py27-firefox as well as py27)
+    local OLD_TOXENV="$TOXENV"
+    unset TOXENV
+    local RUN_ENVS=$(tox -l | grep "^${OLD_TOXENV}\(\-\|\$\)" | tr "\n" ",")
+    if [[ -n "$RUN_ENVS" ]]; then
+        tox -e "$RUN_ENVS"
```

Comment:
So I'm not super happy about running every test N times even when the test doesn't do anything vaugely product-dependent.

Reply:
I don't think that's worth optimising for at this stage when the whole testsuite takes under a second to run.